arrow-fx/src/test/kotlin/arrow/fx/RefTest.kt

```diff
@@ -18,7 +18,7 @@ import io.kotlintest.shouldBe
 class RefTest : UnitSpec() {
 
   init {
-    fun <F> MonadDefer<F>.tests(EQF: EqK<F>, RF: RefFactory<F>): Unit {
```

Comment:
Congrats @danimontoya !! You got it!! Now we could undo these changes. They aren't API but an example for other functions :+1:

Reply:
Sure!

Reply:
Thanks :)